src/app/containers/StoryPromo/index.jsx

```diff
@@ -139,7 +139,7 @@ const StoryPromoContainer = ({
     return null;
   }
 
-  const topStory = promoType === 'top';
+  const topStory = promoType === 'top' || promoType === 'leading';
```

Comment:
Since this will also be applied to leading promos, shall we rename the constant to something like `useLargeImages` and also inside the `StoryPromoImage`?

Reply:
I've updated this :)